src/coverlet.core/Abstracts/IRetryHelper.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Coverlet.Core.Abstracts
+{
+    public interface IRetryHelper
```

Comment:
Are you sure you want to expose such a type in coverlet.core? Have you thought about other alternatives which don't need to expose this, ie internalvisibleto, reflection?

Reply:
Good catch...I prefer internalvisibleto

Reply:
Btw we should review all visibility of core lib to allow in future to expose core directly as package for advanced scenario, this is a first step.

Reply:
Why would you need such a package?

Reply:
Some guys wanted it https://github.com/tonerdo/coverlet/issues/212